### DISCUSSION TITLE
Can I add animation when resizing a window using the shortcuts ?

### DISCUSSION THREAD

**Discussion by julescllt** [rxhanson/Rectangle#1211] [Q&A]
Hi, I apologize if this simple question has already been asked, but I'm not a regular github user and have had trouble finding an answer to it. I'd like to add a small animation duration when resizing my windows using the shortcuts; that is, I don't want my windows to instantly get to the right dimensions (like almost maximize, which I usually go for), but I'd rather like to see a smooth resizing transition.

I found that I can change the animation duration for the snap areas feature but this is not exactly what I'm looking for; I mostly use the keyboard shortcuts and want my _window_ to animate, not the snap area preview.

Best,

**Comment by rxhanson** [comment]
This is the right place to ask. There are macOS limitations that make this something that wouldn’t be a great user experience (choppy and not a nice animation), and that’s why something like this has never been added.

**Reply by julescllt** [reply]
Thanks a lot !
